src/v2/TweetPaginator.ts

```diff
@@ -2,10 +2,6 @@ import { TwitterRateLimit } from '../types';
 import TwitterApiv2ReadOnly from './client.v2.read';
 import { Tweetv2SearchParams, Tweetv2SearchResult } from './types.v2';
 
-export interface PaginatorParams {
```

Comment:
PaginatorParams were created in early stages of TweetPaginator developement, i've forgot to remove it when it becomes useless. Ok to remove it!